### PR TITLE
[warm boot] introduce command line options to warm/fast reboot scripts

### PR DESCRIPTION
**Pull request by yxieca**
**- What I did**
- Add command line option parsing to warm/fast reboot script.
- Allow ignoring orchagent pausing failure with -f option.
- Add verbose output statements.
- Log progress to syslog

### CODE REVIEW

File: scripts/fast-reboot [hunk 1]
```diff
@@ -15,10 +17,48 @@ then
...
+                ;;
+        esac
+    done
+}
+
 sonic_asic_type=$(sonic-cfggen -y /etc/sonic/sonic_version.yml -v asic_type)
 
 function clear_warm_boot()
 {
+    debug "Failure ($?) recovering ..."
```
Comment (qiluo-msft) [inline scripts/fast-reboot]: >recovering [](start = 24, length = 10)

cleanup

File: scripts/fast-reboot [hunk 2]
```diff
@@ -15,10 +17,48 @@ then
     exit 1
 fi
 
+function debug()
+{
+    if [[ x"${VERBOSE}" == x"yes" ]]; then
+        echo `date` $@
```
Comment (qiluo-msft) [inline scripts/fast-reboot]: >echo [](start = 8, length = 4)

This is a script for shutdown. Left message on stdout may be the best choice. How about syslog?

Comment (yxieca) [reply scripts/fast-reboot]: Good idea. I also send these debug output to syslog.

I think you meant to say not leaving messages on stdout. But I think that is what verbose supposed to mean.

**Comment by yxieca** [comment]
retest this please

**Pull request merged**